plugins/minigames.json

```diff
@@ -457,6 +457,12 @@
         }
       ],
       "versions": {
+        "2.0.0": {
+          "api_version": 7,
+          "commit_sha": "88841cd",
+          "released_on": "23-07-2023",
+          "md5sum": "afca5167b564a7f7c0bdc53c3e52d198"
+        },
```

Comment:
uh oh, seems like we updated the collector minigame's version meta, we'll have to remove this entire `2.0.0` object to revert the change, and create `null` meta entry for 2.0.0 version for dodge the ball minigame :cat:

Reply:
check now 😭